app/change_sets/hyrax/collection_change_set.rb

```diff
@@ -36,26 +36,36 @@ def select_files
       Hash[all_files_with_access]
     end
 
-    # Terms that appear above the accordion
-    def primary_terms
-      [:title]
-    end
+    class << self
+      # Combined primary and secondary terms
+      def terms
+        primary_terms + secondary_terms
+      end
 
-    # Terms that appear within the accordion
-    def secondary_terms
-      [:creator,
-       :contributor,
-       :description,
-       :keyword,
-       :license,
-       :publisher,
-       :date_created,
-       :subject,
-       :language,
-       :identifier,
-       :based_near,
-       :related_url,
-       :resource_type]
+      # Terms that appear above the accordion
+      def primary_terms
```

Comment:
You just changed these instance methods to a class methods, why? That's going to mess up the views which are expecting to call this on the instance.

Reply:
Ok thanks, will revert them.